validator/tasks/scheduler.py

```diff
@@ -24,59 +22,69 @@
 
 
 async def _get_models(keypair: Keypair) -> AsyncGenerator[str, None]:
-    response: list[dict[str, Any]] = await call_content_service(csts.GET_RANDOM_MODELS_ENDPOINT, keypair)
-    models = [ModelData(**model) for model in response]
-    random.shuffle(models)
-    for m in models:
-        yield m.model_id
+    response = await call_content_service(cst.GET_RANDOM_MODELS_ENDPOINT, keypair)
+    if not isinstance(response, list):
+        raise TypeError("Expected a list of responses from GET_ALL_MODELS_ENDPOINT")
+    models: list[dict[str, Any]] = response
+    TEMP_MODEL_FAMILIES_ACCEPTED = ["qwen", "llama", "falcon", "mistral", "gemma", "gemini", "phi"]
```

Comment:
These bits were removed - no longer needed 

async def _get_models(keypair: Keypair) -> AsyncGenerator[str, None]:
    response: list[dict[str, Any]] = await call_content_service(csts.GET_RANDOM_MODELS_ENDPOINT, keypair)
    models = [ModelData(**model) for model in response]
    random.shuffle(models)
    for m in models:
        yield m.model_id